components/heartbreaker/control.tsx

```diff
@@ -68,6 +69,10 @@ const Control = () => {
       return;
     }
 
+    if (!gameIsLive && !gameTimer) {
```

Comment:
If game timer hasn't started no bet should be made as the backend isn't done reconciling the previous round.